src/js/common/schemaform/review/StringField.jsx

```diff
@@ -14,13 +14,15 @@ import {
  */
 export default function StringField(props) {
   const { registry, schema, uiSchema, formData } = props;
-  const enumOptions = Array.isArray(schema.enum) && optionsList(schema);
+  const uiOptions = getUiOptions(uiSchema);
+  const labels = uiOptions.labels || {};
+  const enumOptions = Array.isArray(schema.enum) && optionsList(
```

Comment:
Yeah, I realized after sending this that it was a bit silly--changed to follow your suggestion.